bundles/org.openhab.binding.ring/src/main/java/org/openhab/binding/ring/handler/AccountHandler.java

```diff
@@ -0,0 +1,437 @@
+                        updateState(channelUID, new StringType(lastEvents.get(eventIndex).getKind()));
+                    }
+                    break;
+                case CHANNEL_EVENT_DOORBOT_ID:
+                    if (eventListOk) {
+                        updateState(channelUID, new StringType(lastEvents.get(eventIndex).getDoorbot().getId()));
+                    }
+                    break;
+                case CHANNEL_EVENT_DOORBOT_DESCRIPTION:
+                    if (eventListOk) {
+                        updateState(channelUID,
+                                new StringType(lastEvents.get(eventIndex).getDoorbot().getDescription()));
+                    }
+                    break;
+                /*
+                 * case CHANNEL_CONTROL_STATUS:
+                 * updateState(channelUID, status);
+                 * break;
+                 */
+                case CHANNEL_CONTROL_ENABLED:
+                    updateState(channelUID, enabled);
+                    break;
+                default:
+                    logger.debug("Command received for an unknown channel: {}", channelUID.getId());
+                    break;
+            }
+            refreshState();
+        } else if (command instanceof OnOffType) {
+            OnOffType xcommand = (OnOffType) command;
+            switch (channelUID.getId()) {
+                /*
+                 * case CHANNEL_CONTROL_STATUS:
+                 * status = xcommand;
+                 * updateState(channelUID, status);
+                 * break;
+                 */
+                case CHANNEL_CONTROL_ENABLED:
+                    if (!enabled.equals(xcommand)) {
+                        enabled = xcommand;
+                        updateState(channelUID, enabled);
+                        if (enabled.equals(OnOffType.ON)) {
+                            Configuration config = getThing().getConfiguration();
+                            Integer refreshInterval = ((BigDecimal) config.get("refreshInterval")).intValueExact();
+                            ;
+                            startAutomaticRefresh(refreshInterval);
+                        } else {
+                            stopAutomaticRefresh();
+                        }
+                    }
+                    break;
+                default:
+                    logger.debug("Command received for an unknown channel: {}", channelUID.getId());
+                    break;
+            }
+        } else {
+            logger.debug("Command {} is not supported for channel: {}", command, channelUID.getId());
+        }
+    }
+
+    /**
+     * Refresh the state of channels that may have changed by (re-)initialization.
+     */
+    @Override
+    protected void refreshState() {
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing Ring Account handler");
+        super.initialize();
+
+        Configuration config = getThing().getConfiguration();
```

Comment:
Maybe it would be worth declaring a configuration class and then `AccountConfiguration config = getThing().getConfiguration().as(AccountConfiguration.class);` - this would save you from below lines